utils/docker/Dockerfile.leap.42.3

```diff
@@ -17,14 +17,15 @@ ARG UID=1000
 # It's better to put the zypper update in the same "cache layer" as the
 # zypper install command so that the database is updated if/when the
 # installed packages list below is updated
-RUN zypper --non-interactive update
+RUN zypper --non-interactive --no-gpg-checks refresh; \
+    zypper --non-interactive update; \
```

Comment:
You don't want a `; \` at the end of this line as this is the end of the `RUN` command starting on line 20.